struct-field-names-as-array/tests/tests.rs

```diff
@@ -26,6 +26,15 @@ struct TestSkip {
     #[field_names_as_slice(skip)]
     c: String,
 }
+#[derive(FieldNamesAsArray, FieldNamesAsSlice)]
```

Comment:
Hate to nitpick on such a nice PR, but I'd prefer spacing between items to be consistent:

```suggestion

#[derive(FieldNamesAsArray, FieldNamesAsSlice)]
```